### ISSUE TITLE
Create pattern for Compound::Subnav-Menu--Main~Grouped

### ISSUE THREAD

**Issue by kristianserrano** [emory-libraries/Pattern-Library#221]

**Comment by laurenhamel** [comment]
Changing this estimate to a 3 since we discussed removing the search filter from the menu.

**Comment by laurenhamel** [comment]
Adding an update regarding the search filter: When building the Services Directory page, I went ahead and enabled query parameters, so this does allow the page to be pre-search (i.e., by appending `?q=some+search+query` to the page URL). This will make it much easier to implement the search box in this subnavigation menu when the time comes.

**Comment by laurenhamel** [comment]
@kristianserrano, I'm reassigning this issue to me to more evenly distribute points. I'm also thinking we should combine this with #220 and leverage style variations to create the subtle differences between the pattern styles. If you agree with this, then I can close #220 as a duplicate or invalid and just bump the points on this issue a little bit.

Also, what are you thoughts on including the search box in this pattern? I know we had discussed a little while back that we would remove it to simplify the development effort, but given the changes I made in the Services Directory page (see my last comment above), I'm now thinking this would be minimal effort to include. Let me know your thoughts.
